source/reference/connection-string.txt

```diff
@@ -51,6 +51,8 @@ The components of this string are:
        specific database using these credentials after connecting to
        the :program:`mongod` instance.
 
+       If username or password contains an at-sign ("@") or colon (":") then the username and/or password must be URL encoded. At-sign ("@") is replaced with "%40" (3 characters), while colon (":") is replaced with "%3A" (3 characters).
+
```

Comment:
Oh, I see that in an old driver spec file (https://github.com/10gen/specifications/commit/e22cd36938b184f3f98b0780b08ae4321becaac3) , they only talk about "@" and ":".  I'll go ahead and merge this in then.